### PR TITLE
setting mnist acgan example

### PR DESCRIPTION
**Pull request by dfalbel**
Not ready to merge!

### CODE REVIEW

**Comment by jjallaire** [comment]
Awesome, let me know when you think it's ready for merge.

**Comment by dfalbel** [comment]
@jjallaire This takes too long to run on CPU ~4h per epoch so it's hard to tell if it's right. I'm configuring an AWS instance with GPU to test now, so it's almost ready to merge :)

File: R/model.R [hunk 1]
```diff
@@ -487,7 +487,7 @@ normalize_x <- function(x) {
   
   # recurse for lists
   if (is.list(x))
-    return(lapply(x, normalize_input))
+    return(lapply(x, normalize_x))
```
Comment (dfalbel) [inline R/model.R]: @jjallaire is this the correct behavior?

Comment (jjallaire) [inline R/model.R]: Good catch! I just made that change on master.

**Comment by dfalbel** [comment]
@jjallaire After long time testing, I found out that I needed to use a lower leraning rate so it works. This is an issue here: https://github.com/lukedeo/keras-acgan/issues/4

Now this is ready to merge. After 6 epochs I have this:

image

In this example the use GPU is indispensable: it takes 6m per epoch in GPU and 6hours per epoch in the GPU.

**Comment by jjallaire** [comment]
Thanks! Very cool :-)

**Pull request merged**